modules/nixos/security.nix

```diff
@@ -31,6 +31,10 @@
       hostNames = [ "[u348918.your-storagebox.de]:23" ];
       publicKey = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIICf9svRenC/PLKIL9nk6K/pxQgoiFC41wTNvoIncOxs";
     };
+    web01 = {
+      hostNames = [ "web01.nix-community.org" ];
+      publicKey = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIF5CDwgR6N6shJGLRh73EjgwXdGYFOMjBsIRDiy3f+2t";
```

Comment:
This key doesn't match what is generated from `ssh_host_ed25519_key` in `web01/secrets.yaml`.
```
❯ ./inv print-keys --flake-attr web01
###### Public keys ######
ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIBlk4GXei97txlkLtRQDblje0YXZxQnu5w7rVSBPzYRl adisbladis@mochi

###### Age keys ######
age1dg06e2l664lek3het63vrdrvzyrzt2tcf4peellhxc33aj2wf3ysgja8gl

```